metpy/calc/thermo.py

```diff
@@ -232,7 +232,7 @@ def lfc(pressure, temperature, dewpt):
                               direction='increasing')
     # Two possible cases here: LFC = LCL, or LFC doesn't exist
     if len(x) == 0:
-        if np.any(ideal_profile > temperature):
+        if np.any(_less_or_close(ideal_profile[1:], temperature[1:]) == False):
```

Comment:
I think the idea was to use `_less_or_close` *instead* of slicing off the first point.